lib/src/view/game/message_screen.dart

```diff
@@ -231,13 +237,15 @@ class _ChatBottomBarState extends ConsumerState<_ChatBottomBar> {
             hintText: context.l10n.talkInChat,
           ),
           cupertinoDecoration: BoxDecoration(
-            border: Border.all(color: Theme.of(context).colorScheme.outline),
+            border: Border.all(
+              color: CupertinoColors.separator.resolveFrom(context),
+            ),
             borderRadius: borderRadius,
           ),
           controller: _textController,
-          keyboardType: TextInputType.text,
+          keyboardType: TextInputType.multiline,
```

Comment:
I don't think the user should be able to make new lines as lila (or the website frontend) dosen't allow it.

Reply:
yes indeed; in that case `maxLines` should be 1 too

Reply:
I am not sure `maxLines` should be 1. On the one hand, if `maxLines` is greater than one, the user can fully see his message before sending it. On the other hand, it can be confusing for the user. But the return key is not avaible (at least on Android) for `TextInputType.text` so that should help the user to understand that his message is only one line.

Also, it is the website frontend which applies a treatment to text (I think it reduces white spaces to 1 and remove new lines).

Reply:
yes it is a compromise and we should take the approach that is the less confusing for the user; still not sure what is the best actually;

on the website it will remove the new lines, but on mobile it will show them; why not let the user send multiline text on mobile after all?

Reply:
> on the website it will remove the new lines, but on mobile it will show them if we want to; why not let the user send multiline text on mobile after all?

Because if a user sends a message with newlines, it will weird for the receiver on the website as newlines will be removed.
I guess the best solution would be to remove the treatment of text on the website but they surely did this for a reason.

Reply:
I'm pretty sure it's only to keep things simple because you hit <enter> to send the message.
Anyway, even though I'd like to make the best out of the mobile app, we're not a chat application and having a forced 1 line text is already good for a start. 
I'll ask the server team what they think about multiline text on the website.